package/cluster/services/composition.yaml

```diff
@@ -38,3 +38,27 @@ spec:
         # Derive the Prometheus operator image and tag from the XR.
         - fromFieldPath: spec.operators.prometheus.version
           toFieldPath: spec.forProvider.chart.version
+    - name: releaseUXP
+      base:
+        apiVersion: helm.crossplane.io/v1beta1
+        kind: Release
+        spec:
+          rollbackLimit: 3
+          forProvider:
+            namespace: upbound-system
+            chart:
+              # from https://github.com/upbound/universal-crossplane
+              # Note that default values are overridden by the patches below.
+              name: universal-crossplane
+              repository: https://charts.upbound.io/stable
+              version: "1.10.1-up.1"
```

Comment:
do you want to expose UXP version as a user settable config value, similar to how Prometheus version is exposed?

Reply:
@jbw976 I was thinking about it and more inclined toward the scenario of *not* exposing the UXP version.
It will complicate the API(we will have to propagate it up to Cluster abstraction) with no real benefits.
I prefer maintaining the 'hardcoded' UXP version in a single place in this case instead of an example Claim.

Reply:
that's reasonable, thanks for explaining your thinking!